WordPress/src/main/java/org/wordpress/android/localcontentmigration/LocalAccessTokenProviderHelper.kt

```diff
@@ -0,0 +1,11 @@
+package org.wordpress.android.localcontentmigration
+
+import org.wordpress.android.fluxc.store.AccountStore
+import org.wordpress.android.localcontentmigration.LocalContentEntityData.AccessTokenData
+import javax.inject.Inject
+
+class LocalAccessTokenProviderHelper @Inject constructor(
+    private val accountStore: AccountStore,
+): LocalDataProviderHelper {
+    override fun getData(localSiteId: Int?, localEntityId: Int?) = AccessTokenData(token = accountStore.accessToken)
```

Comment:
I'm on the fence about whether this could be tested.. but if we do add a unit test for this helper, maybe we test that `getData` returns the correct `AccessTokenData` (including that it has the correct token, from a mock), but perhaps not that it invokes a particular getter. I.e. state verification instead of behavior verification. Wdyt?

Reply:
I've added a test here: https://github.com/wordpress-mobile/WordPress-Android/pull/17399/commits/2b0607ee5d20ef5f16917a523bf9f4908cf08ecc.